frontend/tests/emailWorkflow.test.ts

```diff
@@ -0,0 +1,100 @@
+import { expect } from '@playwright/test';
+import { test } from './fixtures';
+import { UserDashboardPage } from './pages/userDashboardPage';
+import { loginAs, logout } from './authHelpers';
+import { getInbox } from './mailboxHelpers';
+import { UserAccountSettingsPage } from './pages/userAccountSettingsPage';
+import { ResetPasswordPage } from './pages/resetPasswordPage';
+import { randomUUID } from 'crypto';
+import { LoginPage } from './pages/loginPage';
+
+// TODO: Investigate why this test passes on Firefox but fails on Chromium
+test('register, verify, update, verify email address', async ({ page, tempUser }) => {
+  test.slow(); // Checking email and logging in repeatedly takes time
+  await loginAs(page.request, tempUser.email, tempUser.password);
+  const userDashboardPage = await new UserDashboardPage(page).goto();
+  await userDashboardPage.emailVerificationAlert.assertPleaseVerify();
+
+  // Request extra verification email and verify
+  await userDashboardPage.emailVerificationAlert.clickResendEmail();
+  await userDashboardPage.emailVerificationAlert.assertVerificationSent();
+
+  const inboxPage = await getInbox(page, tempUser.mailinatorId).goto();
+  await expect(inboxPage.emailLocator).toHaveCount(2);
+  let emailPage = await inboxPage.openEmail();
+  let pagePromise = emailPage.page.context().waitForEvent('page');
+  await emailPage.clickVerifyEmail();
+  let newPage = await pagePromise;
+  let userPage = await new UserAccountSettingsPage(newPage).waitFor();
+
+  await userPage.emailVerificationAlert.assertSuccessfullyVerified();
+
+  // Verify verification alert goes away on navigation
+  await userPage.goHome();
+  await userPage.emailVerificationAlert.assertGone();
+
+  // Request new email address
+  const newMailinatorId = randomUUID();
+  const newEmail = `${newMailinatorId}@mailinator.com`;
+  // Was:
+  // const newEmail = `newemail-${tempUser.email}`;
+  // const newMailinatorId = newEmail.replace('@mailinator.com', '');
```

Comment:
It's not obvious to me what these mean

Reply:
Leftover comments during development; I'll get rid of them.

Reply:
Done in commit 3c01d5d8dc56ffb161fc2fad88d69f359b5e3d37.